tool/main.c

```diff
@@ -11,18 +11,18 @@
 static void
 Usage(const char *arg0)
 {
-  fprintf(stderr, "Usage: %s <-c corefile> <-p pid> [-s core_size] [-Pz]\n", arg0);
+  fprintf(stderr, "Usage: %s <-c corefile> <-p pid> [-s core_size] [-z]\n", arg0);
   fprintf(stderr, "<-h>               Show this message and exit\n");
   fprintf(stderr, "<-c core_file>     Collect the core dump into a file core_file\n");
   fprintf(stderr, "<-p pid>           Collect the core dump of process with pid\n");
-  fprintf(stderr, "<-s core_size>     Limit the size of core_file to core_size\n");
-  fprintf(stderr, "<-P>               Prioritize the smaller memory segment over larger segments,\n");
-  fprintf(stderr, "                   will be useful when core_file size is limited with -s\n");
+  fprintf(stderr, "<-s core_size>     Limit the size of core_file to core_size,\n");
+  fprintf(stderr, "                   prioritize smaller segment over larger segments\n");
```

Comment:
this is an implementation detail that I don't think is appropriate for cmdline help

Reply:
Sure, I will remove this...